client/components/FaveButton.tsx

```diff
@@ -0,0 +1,30 @@
+import React from "react";
+import { Fave } from "@prisma/client";
+
+import StarBorderIcon from "@mui/icons-material/StarBorder";
+import Paper from "@mui/material/Paper";
+import IconButton from "@mui/material/IconButton";
+import Typography from "@mui/material/Typography";
+import Stack from "@mui/material/Stack";
+
+type FaveButtonProps = {
+  zoopId: number;
+  faves: Fave[];
+};
+
+const FaveButton = ({ zoopId, faves }: FaveButtonProps) => {
+  return (
+    <>
+      <Paper variant="outlined" sx={{ width: 60, height: "100%" }}>
+        <Stack direction="row" alignItems="center">
```

Comment:
I think everything should be inside a button, that way the whole thing is interactive. it's better for usability and accessibility

Could you add a Stack inside a Button instead, and then put StarBorderIcon and the Typography inside the Stack?